src/Illuminate/Broadcasting/Broadcasters/RedisBroadcaster.php

```diff
@@ -20,16 +20,16 @@ class RedisBroadcaster extends Broadcaster
     /**
      * The Redis connection to use for broadcasting.
      *
-     * @var string
+     * @var ?string
      */
-    protected $connection;
+    protected $connection = null;
```

Comment:
This is the default in the constructor, which is bypassed by mocks. The same goes for `$prefix`.